brambling/templates/brambling/event/order/attendee_basic_data.html

```diff
@@ -33,57 +33,123 @@ <h3 class="margin-leader-0">
 					New Attendee
 				{% endif %}
 			</h3>
-			
-			{% include "brambling/forms/_top_error_list.html" %}
+
+			{% if basic_data_form.errors %}
+				{% include "brambling/forms/_top_error_list.html" with form=basic_data_form %}
+			{% elif housing_form.errors %}
+				{% include "brambling/forms/_top_error_list.html" with form=housing_form %}
+			{% endif %}
```

Comment:
no, it is, but this was the easiest way to only display the top error list once. (They don't have non-field errors.)